examples/.polaris.yaml

```diff
@@ -6,7 +6,7 @@ checks:
   pullPolicyNotAlways: ignore
   readinessProbeMissing: danger
   livenessProbeMissing: danger
-  metadataAndNameMismatched: danger
```

Comment:
@debu99 i have a feeling on the older version of Polaris `metadataAndNameMismatched` didn't exist and was not being caught. On the newer version it seems to error out when I keep this in the config. So I switched to: `metadataAndInstanceMismatched` and that throws a danger /error so i went ahead and added `metadataAndInstanceMismatched` as a warning. I checked the Polaris docs on the fairwinds site and it seems like `metadataAndInstanceMismatched` is the correct key to use not `metadataAndNameMismatched`. Curious to know your thoughts and whether we should roll back what I've done here.